evap/staff/importers.py

```diff
@@ -498,25 +498,33 @@ def write_enrollments_to_db(self, semester, vote_start_datetime, vote_end_date):
             for evaluation, participants in participants_per_evaluation.items():
                 evaluation.participants.add(*participants)
 
-        msg = format_html(
-            _("Successfully created {} courses/evaluations, {} participants and {} contributors:"),
-            len(self.evaluations),
-            len(students_created),
-            len(responsibles_created),
-        )
-        msg += create_user_list_html_string_for_message(students_created + responsibles_created)
+            msg = format_html(ngettext('Successfully created one course/evaluation,', 'Successfully created {evaluation} courses/evaluations,', len(self.evaluations),).format(evaluation = len(self.evaluations)))
+            if len(students_created) == 0 and len(responsibles_created) == 0:
+                msg += format_html(_(" no students and no contributors."))
+            else:
+                if len(students_created) == 0:
+                    msg += format_html(_(" no students"))
+                else:
+                    msg += format_html(ngettext(' one student', ' {users} students', len(students_created),).format(users = len(students_created)))
+                    if len(responsibles_created) == 0:
+                        msg += format_html(_(" and no contributors:"))
```

Comment:
mismatch between code name (responsibles) and text name (contributors). One should be fixed (but if you don't want to, you don't have to).